src/test/java/pl/bugdemons/temp/config/ConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+package pl.bugdemons.temp.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class ConfigurationTest {
+    @Test
+    public void configWithoutInitShouldBeEmpty() {
+        var selenideProperties = getSelenidePropertiesFromSystem();
```

Comment:
wydaje mi się, że to powinno sprawdzać czy niema jakichkolwiek propertisów + osobne 2 testy na propertisy selenide (empty/not empty)

Reply:
ale config jest ustawiony na systemowe propertiesy, więc tam zawsze coś będzie, raczej nie będziemy ich czyścić :P 
Można przyjąć inną koncepcje i nie ustawiać tego na poziomie systemu, ale to wydaje mi się zapewnia największą dostępność i reużywalność.